### PR TITLE
feat(taxonomy): added metadata taxonomy field support

### PR DESCRIPTION
**Pull request by TylerGauntlett**
* Added support for new taxonomy fi

eld (if returned by the backend)

### CODE REVIEW

File: src/api/Metadata.js [hunk 1]
```diff
@@ -1111,10 +1111,10 @@ class Metadata extends File {
         }
 
         const url = this.getMetadataOptionsUrl(scope, templateKey, fieldKey);
-        const { marker, searchInput, signal } = options;
+        const { marker, searchInput: query_text, signal } = options;
         const params = {
             ...(marker ? { marker } : {}),
-            ...(searchInput ? { searchInput } : {}),
+            ...(query_text ? { query_text } : {}),
```
Comment (jmcbgaston) [inline src/api/Metadata.js:1117]: nit but should this be camel cased?

**Review by jmcbgaston — APPROVED** [review]
LGTM

File: src/api/Metadata.js (hunk 1 above)
Comment (TylerGauntlett) [reply src/api/Metadata.js:1117]: This is intentional as the query string param is required in snake case (ie `/metadata?query_text=search`)

**Review by JChan106 — APPROVED** [review]
lgtm

File: src/elements/content-sidebar/fetchers/metadataTaxonomyFetcher.ts
Comment (tjuanitas) [inline src/elements/content-sidebar/fetchers/metadataTaxonomyFetcher.ts:1]: this repo favors `utils` folder path

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**